cli/src/main.rs

```diff
@@ -149,10 +149,11 @@ fn verify_signature(signed_file: &PathBuf) -> Result<()> {
     let signed_json = deserialize_from_file::<Value>(signed_file)?;
     match determine_file_type(&signed_json)? {
         FileType::Certificate => {
-            gcert::validate_certificate(signed_json).map(|result| println!("{:?}", result))
+            Ok(gcert::validate_certificate(signed_json).map(|result| println!("{:?}", result))?)
```

Comment:
```suggestion
            gcert::validate_certificate(signed_json)
                .map(|result| println!("{:?}", result))
                .map_err(Into::into)
```

I think `result.map_err(Into::into)` better communicates what is happening, `Ok(result?)` is mysterious unless you realize that the point is to transform the error.